src/components/Addbook.jsx

```diff
@@ -73,15 +73,15 @@ const Div = styled.div`
     .title {
       height: 2.813rem;
       border-radius: 4px;
-      border: solid 1px #2e90f0;
+      border: solid 1px #e1e1e1;
```

Comment:
- as seen below there should be some padding between the cards and the navbar
- as seen below the background of your books are different from the original design please fix this 👍🏾 
- your  book should have padding around the body of the book as seen below
- please remove the border around the see progress button 
- please use the correct font for this project which is `Roboto, Helvetica, sans-serif

your design
![image](https://github.com/shihabrafio/BookStore/assets/60612329/66180b81-7bd9-437e-a663-09a71405bfb1)

required design
![image](https://github.com/shihabrafio/BookStore/assets/60612329/1e02dc5d-8bba-469d-b8c8-6bcde5e7eb5d)